app/assets/stylesheets/components/_result-item.scss

```diff
@@ -11,3 +11,20 @@
   border-left: 5px solid govuk-colour("blue");
   padding: govuk-spacing(3);
 }
+
+// stylelint-disable declaration-no-important
+@include govuk-media-query($media-type: print) {
+  .app-c-result-item  {
+    break-inside: avoid;
+    border-color: $govuk-print-text-colour !important;
```

Comment:
Is the `!important` needed here?

Reply:
I thought it was, but turns out not so I've deleted that one.